### PR TITLE
SkiaLayerAnalytics - send events for different steps of rendering

### PR DESCRIPTION
**Pull request by igordmn**
It will help us to detect GPU's, on which we have crashes. After we compare two events, and if in the second event we have less hits, it means we have a crash. We can investigate it, and if the issue only in GPU, we will add it to not-supported-adapter.list

The plan to integrate this analytics to Toolbox.

### CODE REVIEW

File: skiko/src/jvmMain/cpp/windows/directXRedrawer.cc [hunk 1]
```diff
@@ -19,18 +19,6 @@
...
-// with the current Swing/Skia integration.
-// If PC has other graphics cards suitable for DirectX12, one of them will be used. Otherwise,
-// rendering will falls back to OpenGL.
-const std::vector<std::wstring> notSupportedAdapters{
-    L"Intel(R) HD Graphics 520",
-    L"Intel(R) HD Graphics 530",
-    L"Intel(R) HD Graphics 4400",
-    L"NVIDIA GeForce GTX 750 Ti",
-    L"NVIDIA GeForce GTX 960M",
-    L"NVIDIA Quadro M2000M"};
```
Comment (dima-avdeev-jb) [inline skiko/src/jvmMain/cpp/windows/directXRedrawer.cc:32]: Are we skip this list of video adapters?
Or it moves to another place?

**Review by dima-avdeev-jb — APPROVED** [review]

File: skiko/src/jvmMain/cpp/windows/directXRedrawer.cc (hunk 1 above)
Comment (igordmn) [reply skiko/src/jvmMain/cpp/windows/directXRedrawer.cc:32]: It is [here](https://github.com/JetBrains/skiko/blob/master/skiko/src/jvmMain/resources/not-supported-adapter.list)

Comment (dima-avdeev-jb) [reply skiko/src/jvmMain/cpp/windows/directXRedrawer.cc:32]: Also, we have broken build on android

**Comment by dima-avdeev-jb** [comment]
Also, we have broken build on android

**Comment by dima-avdeev-jb** [comment]
Now, we have 2 failed tests on Windows:
org.jetbrains.skiko.SkiaLayerPerformanceTest[awt] > FPS is near display refresh rate (multiple windows)[awt] FAILED
org.jetbrains.skiko.SkiaLayerTest[awt] > analytics[awt] FAILED

**Comment by dima-avdeev-jb** [comment]
Cool, all CI checks work's Fine!

**Pull request merged**